kimminwoo/likelion-assignment-crud/src/main/java/com/kimminwoo/likelionassignmentcrud/student/application/StudentService.java

```diff
@@ -0,0 +1,45 @@
+package com.kimminwoo.likelionassignmentcrud.student.application;
+
+
+import com.kimminwoo.likelionassignmentcrud.student.api.dto.request.StudentSaveRequestDto;
+import com.kimminwoo.likelionassignmentcrud.student.api.dto.response.StudentInfoResponseDto;
+import com.kimminwoo.likelionassignmentcrud.student.api.dto.response.StudentListResponseDto;
+import com.kimminwoo.likelionassignmentcrud.student.domain.Student;
+import com.kimminwoo.likelionassignmentcrud.student.domain.repository.StudentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class StudentService {
+    private final StudentRepository studentRepository;
+
+    @Transactional
+    public void studentSave(StudentSaveRequestDto studentSaveRequestDto) {
+        Student student = Student.builder()
+                .name(studentSaveRequestDto.name())
+                .phoneNumber(studentSaveRequestDto.phoneNumber())
+                .build();
+        studentRepository.save(student);
+    }
+
+    public StudentListResponseDto studentFindAll() {
+        List<Student> students = studentRepository.findAll();
+
+        List<StudentInfoResponseDto> studentInfoResponseDtoList = students.stream()
+                .map(StudentInfoResponseDto::from)
+                .toList();
+        return StudentListResponseDto.from(studentInfoResponseDtoList);
+    }
+
+    public StudentInfoResponseDto studentFindOne(Long studentId) {
+        Student student = studentRepository
+                .findById(studentId)
+                .orElseThrow(IllegalArgumentException::new);
```

Comment:
```suggestion
                .orElseThrow(() -> new IllegalArgumentException("해당 학생을 찾을 수 없습니다. ID: " + studentId));

```
위와 같이 구체적인 오류 메시지를 작성하면, 실제로 디버깅 시 어떤 문제로 오류가 발생했는지 빠르게 파악할 수 있어 매우 도움이 됩니다!

만약 모든 예외가 단순히 `IllegalArgumentException`만 던지고 메시지가 없다면,
여러 메서드 중 어느 위치에서 문제가 발생했는지를 파악하기가 어려워져요.
결과적으로 문제 수정에 더 많은 시간이 걸릴 수 있습니다.

다른 메소드나 클래스에서도 이런 방식으로 구체적인 메시지를 작성해주세요!